.github/workflows/complement.yml

```diff
@@ -0,0 +1,36 @@
+# This task does not run complement tests, see tests.yaml instead.
```

Comment:
would be tempted to rename the file so it's more obvious that this is only for uploading the complement image.
`push_complement_image.yml` would be fine IMO.

Reply:
renamed per suggestion.